verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -245,8 +246,8 @@ class BulkCaseForm extends React.Component<
                 name: 'hospitalAdmission',
                 dateRange: c.dateHospitalized
                     ? {
-                          start: c.dateHospitalized,
-                          end: c.dateHospitalized,
+                          start: toUTCDate(c.dateHospitalized),
```

Comment:
nit: this is actually more like `toUTCDateString`
hyper-nit: why `toString()` here and `toISOString()` above? I'd personally rather use `toISOString()`

Reply:
BTW this is also how we do it in single case entry form.

Essentially, we need to display all the dates in UTC, but no one is concerned with timezone when they are entering the data. So if I type in a date and then use `.toISOString()` it may change the date value because it's assuming I intentionally used my own time zone. So `toUTCDate` changes the date to be in the UTC timezone, but keep the value of the date.

I agree it's unfortunate but this is what we've come up with so far. I suppose you are right on the naming, but it's used in a bunch of places that I'd rather not change, at least not in this PR :)

Reply:
(FYI cmd+shift+f > replace is your friend for multiple files replacements in vscode)